app/controllers/subjects_controller.rb

```diff
@@ -0,0 +1,51 @@
+class SubjectsController < ApplicationController
+  before_action :find_by_subject, only: %i(edit destroy)
+  before_action :logged_in_user
+
+  def index
+    @subjects = Subject.includes(:user).all.page(params[:page]).per Settings.s
```

Comment:
Em includes vào để làm gì đấy

Reply:
"Tải các bản ghi liên quan trước" ở đây anh không thấy bản ghi liên quan nào phải tải trước cả, mục đích đơn giản là em chỉ có in ra toàn bộ các bản ghi của subject thôi 
"giới hạn số lượng truy vấn SQL" hiện tại em đang chỉ load ra 5 bản ghi khi dùng kaminari mà 
Đâu cần phải  dùng includes ở chỗ này

Reply:
Em phải bảo với anh là ở view em gọi user_name nên cần load trước user chứ